includes/widgets/alert.php

```diff
@@ -287,6 +323,127 @@ protected function register_controls() {
 
 		$this->end_controls_section();
 
+		$this->start_controls_section(
+			'section_dismiss_icon',
+			[
+				'label' => esc_html__( 'Dismiss Icon', 'elementor' ),
+				'tab' => Controls_Manager::TAB_STYLE,
+				'condition' => [
+					'show_dismiss' => 'show',
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'dismiss_icon_size',
+			[
+				'label' => esc_html__( 'Size', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 100,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}}' => '--dismiss-icon-size: {{SIZE}}{{UNIT}};',
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'dismiss_icon_vertical_position',
+			[
+				'label' => esc_html__( 'Vertical Position', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'range' => [
+					'px' => [
+						'min' => -100,
+						'max' => 100,
+					],
+				],
+				'size_units' => [ '%', 'px' ],
+				'selectors' => [
+					'{{WRAPPER}}' => '--dismiss-icon-vertical-position: {{SIZE}}{{UNIT}};',
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'dismiss_icon_horizontal_position',
+			[
+				'label' => esc_html__( 'Horizontal Position', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'range' => [
+					'px' => [
+						'min' => -100,
+						'max' => 100,
+					],
+				],
+				'size_units' => [ '%', 'px' ],
+				'selectors' => [
+					'{{WRAPPER}}' => '--dismiss-icon-horizontal-position: {{SIZE}}{{UNIT}};',
+				],
+			]
+		);
+
+		$this->start_controls_tabs( 'dismiss_icon_colors' );
+
+		$this->start_controls_tab( 'dismiss_icon_normal_colors', [
+			'label' => esc_html__( 'Normal', 'elementor' ),
+		] );
+
+		$this->add_control(
+			'dismiss_icon_normal_color',
+			[
+				'label' => esc_html__( 'Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}}' => '--dismiss-icon-normal-color: {{VALUE}};',
+				],
+			]
+		);
+
+		$this->end_controls_tab();
+
+		$this->start_controls_tab( 'dismiss_icon_hover_colors', [
+			'label' => esc_html__( 'Hover', 'elementor' ),
+		] );
+
+		$this->add_control(
+				'dismiss_icon_hover_color',
+				[
+					'label' => esc_html__( 'Color', 'elementor' ),
+					'type' => Controls_Manager::COLOR,
+					'selectors' => [
+						'{{WRAPPER}}' => '--dismiss-icon-hover-color: {{VALUE}};',
+					],
+				]
+		);
+
+		$this->add_control(
+			'dismiss_icon_hover_transition_duration',
+			[
+				'label' => esc_html__( 'Transition Duration', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'selectors' => [
+					'{{WRAPPER}}' => '--dismiss-icon-hover-transition-duration: {{SIZE}}ms',
+				],
+				'range' => [
```

Comment:
I usually add a default value: 
https://github.com/marcelementor/elementor-pro/blob/d7785f3291c77a2e0553fa15fad666b8b838c8e0/modules/posts/widgets/portfolio.php#L365

Reply:
@hein-obox Same a above.

Reply:
@louiswol94 
From what Aviv says, I think it is best to add the default value for this control (transition duration), but not for the other controls.

Reply:
@hein-obox Done.

Reply:
It it possible set the unit to 's'
Default to 0.3
Steps to 0.1

See: https://github.com/marcelementor/elementor-pro/blob/d7785f3291c77a2e0553fa15fad666b8b838c8e0/modules/posts/widgets/portfolio.php#L365

Reply:
@hein-obox Done.